### DISCUSSION TITLE
Archived discussions 🗃️

### DISCUSSION THREAD

**Discussion by FredrikAugust** [open-source-ideas/ideas#301] [General]
See the old [meta](https://github.com/open-source-ideas/meta/issues) repository for old discussions.

**Comment by KOLANICH** [comment]
Just move them here.

**Reply by FredrikAugust** [reply]
It appears the overflow menu on the issues doesn't work. I'll try again tomorrow if I remember. 

Good point!

**Reply by KaKi87** [reply]
Hello, any news on this ? Thanks

**Reply by FredrikAugust** [reply]
![Screenshot 2021-12-13 at 13 38 25](https://user-images.githubusercontent.com/7691547/145814058-7bc3d401-1445-435b-8284-dd36f71af6f0.png)

It doesn't appear to work, @KaKi87, so I can't transfer them here. Might be because the repository is archived.

**Reply by FredrikAugust** [reply]
If there is a specific point you would like to discuss, you could create a discussion here and just copy the old issue.

**Reply by KaKi87** [reply]
How about temporarily de-archiving the repo while transferring issues ?

**Reply by FredrikAugust** [reply]
I definitely could, but they are better suited as discussions here (IMO), so I don't feel like spending time on that right now. 

Again, if there's something to be discussed, just copy or link to the old issue and create a new discussion here.

If there's a lot of demand for moving all of them properly, then I will definitely do it.

**Reply by KaKi87** [reply]
open-source-ideas/ideas#59 has already been re-created once as open-source-ideas/meta#5 and re-discussed at open-source-ideas/ideas#208, it's already messy enough...

**Reply by FredrikAugust** [reply]
I would say that this particular case is special. 

There should only be one issue, but I have left the last one as it sparked a quite lengthy discussion. 

I don't think creating a discussion to discuss what features the website should implement, how they should be implemented etc. would hurt.

**Reply by FredrikAugust** [reply]
https://github.com/open-source-ideas/ideas/discussions/307
